src/console/UserController.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace bizley\podium\console;
+
+use yii\console\Controller;
+use bizley\podium\Podium;
+use bizley\podium\models\User;
+
+/**
+ * Podium command line interface to RBAC managment
+ * 
+ * @author pavlm
+ */
+class UserController extends Controller
+{
+
+    /**
+     * Changes forum user role.
+     * @param int|string $idOrEmail internal podium user id or email
+     * @param string $role one of 'podiumUser', 'podiumModerator', 'podiumAdmin' or other application defined role 
+     */
+    public function actionAssignRole($idOrEmail, $role)
+    {
+        if (!$user = $this->findUser($idOrEmail)) {
+            return self::EXIT_CODE_ERROR;
+        }
+        $rbac = Podium::getInstance()->getRbac();
+        if (!$role = $rbac->getRole($role)) {
+            $this->stderr('No such role.' . PHP_EOL);
+            return self::EXIT_CODE_ERROR;
+        }
+        if (strpos($role->name, 'podium') === 0) {
+            // remove another podium role
+            $userRoles = $rbac->getRolesByUser($user->id);
+            $podiumRoles = array_filter($userRoles, function ($role) {
+                return strpos($role->name, 'podium') === 0;
+            });
+            foreach ($podiumRoles as $podiumRole) {
+                $rbac->revoke($podiumRole, $user->id);
+            }
+        }
+        $rbac->assign($role, $user->id);
+    }
+
+    /**
+     * Revokes forum user role.
+     * @param int|string $idOrEmail internal podium user id or email
+     * @param string $role
+     */
+    public function actionRevokeRole($idOrEmail, $role)
+    {
+        if (!$user = $this->findUser($idOrEmail)) {
+            return self::EXIT_CODE_ERROR;
+        }
+        $rbac = Podium::getInstance()->getRbac();
+        if (!$role = $rbac->getRole($role)) {
+            $this->stderr('No such role.' . PHP_EOL);
+            return self::EXIT_CODE_ERROR;
+        }
+        $rbac->revoke($role, $user->id);
```

Comment:
In case of revoking Podium role you need to check if at least `Rbac::ROLE_USER` is assigned.